service/options_test.go

```diff
@@ -43,7 +43,7 @@ func TestAddModules_Errors(t *testing.T) {
 }
 
 func TestWithLogger_OK(t *testing.T) {
-	logger := ulog.Logger()
+	logger := ulog.Builder().Build()
```

Comment:
looks like we can provide a package level `Build()` function which will return an empty built builder with no parameters. Would reduce the typing in all the tests and things.